packages/app-mobile/root.tsx

```diff
@@ -1041,20 +1042,18 @@ class AppComponent extends React.Component {
 
 		const mainContent = (
 			<View style={{ flex: 1, backgroundColor: theme.backgroundColor }}>
-				<Drawer
-					// Need to reset the key here based on menu position, otherwise
-					// the drawer will flash open on screen and close every time the
-					// drawer position switches (i.e. when opening or closing a note)
-					key={`main-drawer-${menuPosition}`}
-					open={this.props.showSideMenu}
-					onOpen={() => this.sideMenu_change(true)}
-					onClose={() => this.sideMenu_change(false)}
-					drawerPosition={menuPosition}
-					swipeEdgeWidth={15}
-					drawerStyle={{
-						width: this.state.sideMenuWidth,
+				<SideMenu
+					menu={sideMenuContent}
+					edgeHitWidth={20}
```

Comment:
`edgeHitWidth` was originally `5`, however, on my devices, this doesn't seem large enough to allow opening the menu with touch. As such, it was increased to 20.